### DISCUSSION TITLE
Add an API to publish to GitHub Marketplace

### DISCUSSION THREAD

**Discussion by Th3S4mur41** [community/community#69630] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I use [semantic release](https://github.com/semantic-release/semantic-release) to automate the versioning of actions I maintain.
It also automatically creates the [Tag](https://github.com/semantic-release/git) and the [Release on GitHub](https://github.com/semantic-release/github) with its plugins.

What is missing is the ability to automatically publish the newly created release to [GitHub Marketplace](https://github.com/marketplace).
This is not a semantic release limitation though, but rather a GitHub limitation since [this action is only available from the GitHub UI](https://docs.github.com/en/apps/publishing-apps-to-github-marketplace/using-the-github-marketplace-api-in-your-app/rest-endpoints-for-the-github-marketplace-api), but [not through the API](https://docs.github.com/en/apps/publishing-apps-to-github-marketplace/using-the-github-marketplace-api-in-your-app/rest-endpoints-for-the-github-marketplace-api).

It would be great advantage for automated releases of actions if this were made possible through the API too.

See also: https://github.com/semantic-release/github/issues/629

**Comment by ghostinhershell** [comment]
Hey there @Th3S4mur41, thanks so much for sharing this feature request with our community 🚀
